securedrop/journalist_app/__init__.py

```diff
@@ -28,6 +29,21 @@ def create_app(config):
     CSRFProtect(app)
     Environment(app)
 
+    if config.DATABASE_ENGINE == "sqlite":
+        db_uri = (config.DATABASE_ENGINE + ":///" +
+                  config.DATABASE_FILE)
+    else:
+        db_uri = (
+            config.DATABASE_ENGINE + '://' +
+            config.DATABASE_USERNAME + ':' +
+            config.DATABASE_PASSWORD + '@' +
+            config.DATABASE_HOST + '/' +
+            config.DATABASE_NAME
+        )
```

Comment:
We can definitely get rid of the `else` part since everyone is using SQLite - also we've recently merged in raw SQL statements that are SQLite specific. Not required for merge since your diff here is just moving the `db_uri` logic unchanged into `create_app`, just noting